contracts/liquidityMining/KyberFairLaunchV2.sol

```diff
@@ -586,15 +626,16 @@ contract KyberFairLaunchV2 is IKyberFairLaunchV2, PermissionAdmin, ReentrancyGua
     if (_value > _getBlockTime()) _value = _getBlockTime();
   }
 
-  /**
-   * @dev Call locker contract to lock rewards
-   */
-  function _lockReward(IERC20Ext token, address _account, uint256 _amount) internal {
-    uint256 value = token == IERC20Ext(0) ? _amount : 0;
-    rewardLocker.lock{ value: value }(token, _account, _amount);
+  function _getBlockTime() internal virtual view returns (uint32) {
+    return uint32(block.timestamp);
   }
 
-  function _getBlockTime() internal view virtual returns (uint32) {
-    return uint32(block.timestamp);
+  function _isSameVestingDuration(uint256[] calldata _pids) private view returns (bool) {
+    uint256 val = poolInfo[_pids[0]].vestingDuration;
+    for (uint256 i = 1; i < _pids.length; i++) {
+      if(poolInfo[_pids[i]].vestingDuration == val) continue;
```

Comment:
cleaner if u swap the logic.
`if (poolInfo[_pids[i]].vestingDuration != val) return false;`

Reply:
oh thank you, i will fix that